pkg/resources/genericPool.go

```diff
@@ -55,9 +59,9 @@ func (rp *genericResourcePool) GetEnvs(resourceName string, deviceIDs []string)
 		if i == lastIndex {
 			break
 		}
-		values += " "
+		values += ";"
```

Comment:
OK so we are going back to the old format too. Good to know since I needed to patch kubevirt for the spaced format, now I need to revert back.

Reply:
@booxter 
We need to replace periods  with something, as they are not acceptable in shell variable names anyway. http://tldp.org/LDP/abs/html/gotchas.html 

Spaces are not good choice as delimiters in Env Variable. In shell spaces are fine as value but no in Env. I have seen some issues already. Needed to change this regardless of the naming scheme.

Reply:
I agree we need to replace dots for exactly the reason you mentioned. I haven't seen particular issues with spaces in my testing and kubevirt integration though I agree regardless that it makes sense to revert back to commas or semicolons. Though if I were you I would pick commas since that's what we used before multiendpoints branch was merged. While the updated DP is not in general backwards compatible with the previous behavior, it could help some consumers if you stick to the previous delimiter. For example, see this patch in kubevirt that we needed to adopt to the new space splitting: https://github.com/kubevirt/kubevirt/pull/1692/commits/e370ef884190a51f80f354dcdc5bd455612fea10 If you go with ; instead of , we'll need a similar patch. If you go with comma, we don't need any change.

Reply:
+1 on changing the semicolon back to comma which is used in release-v1 as well.
btw, thanks for the shell scripting gotchas link!

Reply:
> +1 on changing the semicolon back to comma which is used in release-v1 as well.
> btw, thanks for the shell scripting gotchas link!

Ok, we will use "'," instead.